Assets/Controllers/MouseInputManager.cs

```diff
@@ -9,7 +10,8 @@ public class MouseInputManager : MonoBehaviour {
 
     HexMap hexMap;
     GameManager gameManager;
-    bool isDraggingFromPawn;
+    bool isDraggingFromPawn = false;
```

Comment:
Same comment as before, might want to move these into a constructor or explicitly set them.